ratpack-manual/src/content/chapters/13-http.md

```diff
@@ -513,4 +513,128 @@ Support for rendering different representations of a resource (JSON/XML/HTML, GI
 
 ## Sessions
 
-TODO introduce `ratpack-sessions` library
+Whenever you need to maintain consistency between multiple calls by the same client (in other words: stateful operations) you may want to use sessions.
+Ratpack offers a module to handle session handling for you.
+You can add the session module to your project and start using ratpack managed sessions.
+
+### Preparation
+First off you need to add the required dependency to your project.
+Using gradle you can add the dependency by adding `compile group: 'io.ratpack', name: 'ratpack-session'` to dependencies.
+When you just started out your gradle dependency section will look like this:
+
+```language-java
+dependencies {
+  runtime 'org.slf4j:slf4j-simple:1.7.25'
+  compile group: 'io.ratpack', name: 'ratpack-session', version: '1.5.0-rc-2' // note: the version is subject to change
+
+  testCompile "org.spockframework:spock-core:1.0-groovy-2.4"
+}
+```
+
+Don't forget to load the module in ratpack. 
+```language-java
+	bindings {
+		module(SessionModule)
+	}
+```
+
+### Use session
+
+You are now set for sessions.
+The following is a simple example of an application that uses sessions.
+
+```language-java
```

Comment:
Can you add `tested` after `language-java` it should then test that the code is correct in the manual.